src/__tests__/asyncHook.test.ts

```diff
@@ -1,6 +1,11 @@
+import { setDefaultWaitOptions } from 'core'
 import { useState, useRef, useEffect } from 'react'
 
 describe('async hook tests', () => {
+  beforeEach(() => {
+    setDefaultWaitOptions({ interval: 50, timeout: 1000 })
+  })
```

Comment:
My only hesitation is that there isn't a test that uses the _actual_ default options anymore.  We've used the technique of creating seperate files to test things with global side effects so I'd suggest introducing `asyncHook.customDefaults.ts` (or something like that) with this and the tests you've added and keep this file using the regular old defaults.